docs/DeveloperGuide.md

```diff
@@ -621,24 +621,99 @@ testers are expected to do more *exploratory* testing.
    1. Re-launch the app by double-clicking the jar file.<br>
        Expected: The most recent window size and location is retained.
 
-1. _{ more test cases …​ }_
+### Adding a member
 
-### Deleting a Member
+1. Adding a member 
+   1. Prerequisites: No duplicate members exist.
 
-1. Deleting a Member while all members are being shown
+   1. Test case: `addMember Alice p/12345678 e/Alice@gmail.com`<br>
+      Expected: Member is successfully added, with the details of the member displayed in the status bar.
 
-   1. Prerequisites: List all members using the `viewMembers` command. Multiple members in the list. A member with the name of `Alice` exists in the list. No member with the name `Barry` exists in the list.
+   1. Test case: `addMember Bryan p/45678910 e/Bryan@gmail.com r/President`<br>
+      Expected: Member is successfully added, with the details of the member displayed in the status bar.
+      
+   1. Test case: `addMember Alice p/23456789 e/Alice@hotmail.com r/Member`<br>
+      Expected: No person is added since a member with the name Alice already exists. Error details shown in the status bar.
```

Comment:
- Would it be better if it is 'no member is added since...'? Same for the next few test cases. 
- Also, since the test cases are standalone test cases, perhaps you should state the assumption that alice already exists in the list?

Reply:
Removed the duplicate test case

Reply:
Changed the rest